.env.base

```diff
@@ -68,6 +68,7 @@ REACT_APP_FEATURE_LIFI_SWAP=true
 REACT_APP_FEATURE_THOR_SWAP=true
 REACT_APP_FEATURE_THOR_SWAP_STREAMING_SWAPS=true
 REACT_APP_FEATURE_ZRX_SWAP=true
+REACT_APP_FEATURE_JUPITER=false
```

Comment:
```suggestion
REACT_APP_FEATURE_JUPITER_SWAP=false
```

Reply:
Done in [e4891d6](https://github.com/shapeshift/web/pull/8120/commits/e4891d6256da07fae14a9f7eaa0e89ba7aefbe0d)